test/jdk/java/util/Random/RandomTest.java

```diff
@@ -410,4 +413,36 @@ public void testUnsizedDoublesCountSeq() {
         assertEquals(counter.sum(), size);
     }
 
+    /**
+     * Test shuffling a list with Random.from()
+     */
+    public void testShufflingList() {
+        final ArrayList listTest = new ArrayList<Integer>();
```

Comment:
```suggestion
        final var listTest = new ArrayList<Integer>();
```
Using `ArrayList<Integer>` is fine too, but right now this is a raw type.